plugins/generator-1.19.4/forge-1.19.4/procedures/entity_get_max_fall_distance.java.ftl

```diff
@@ -0,0 +1 @@
+(${input$entity}.getMaxFallDistance())
```

Comment:
This should have a int marker (`/*@int*/`) at the very beginning (same in 1.20.1)

Reply:
Done